From fec4b148d524db71605ddb2b4b845ef0caf0fc92 Mon Sep 17 00:00:00 2001
From: Pafcholini <nadyaivanova14@gmail.com>
Date: Sun, 24 Jul 2016 19:32:20 +0200
Subject: [PATCH 1/4] default.prop

Signed-off-by: Pafcholini <nadyaivanova14@gmail.com>
---
 pe/default.prop | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/pe/default.prop b/pe/default.prop
index e50d093..1f8e728 100644
--- a/pe/default.prop
+++ b/pe/default.prop
@@ -11,10 +11,10 @@ persist.radio.lte_vrte_ltd=1
 persist.radio.apm_sim_not_pwdn=1
 persist.radio.add_power_save=1
 persist.data.netmgrd.qos.enable=false
-persist.security.ams.enforcing=1
-ro.secure=1
+persist.security.ams.enforcing=0
+ro.secure=0
 ro.allow.mock.location=0
-ro.debuggable=0
+ro.debuggable=1
 ro.adb.secure=1
 ro.zygote=zygote32
 dalvik.vm.image-dex2oat-Xms=64m
@@ -23,8 +23,10 @@ dalvik.vm.dex2oat-Xms=64m
 dalvik.vm.dex2oat-Xmx=512m
 ro.dalvik.vm.native.bridge=0
 debug.atrace.tags.enableflags=0
-persist.sys.usb.config=mtp
+persist.sys.usb.config=mtp,adb
 ro.smps.gain.spk=0.8
+ro.securestorage.support=false
+wlan.wfd.hdcp=disable
 #
 # BOOTIMAGE_BUILD_PROPERTIES
 #
-- 
2.5.0

