From 3a77f081dc03dc1dcb3f3ec4d434e084fd8cd536 Mon Sep 17 00:00:00 2001
From: pafcholini <nadyaivanova14@gmail.com>
Date: Sun, 30 Oct 2016 18:59:36 +0100
Subject: [PATCH] init.rc

Signed-off-by: pafcholini <nadyaivanova14@gmail.com>
---
 p/init.rc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/p/init.rc b/p/init.rc
index fc8c391..c86e653 100755
--- a/p/init.rc
+++ b/p/init.rc
@@ -1869,3 +1869,9 @@ on property:service.vpnclientd.enable=1
 on property:service.vpnclientd.enable=0
     stop vpnclientd
 
+# init Emotion Kernel
+service emotioninit /sbin/emotion.sh
+    class main
+    user root
+    oneshot
+ 
-- 
2.9.3

