From 5a4f850e6206604f044146e243dffb09c278c74f Mon Sep 17 00:00:00 2001
From: Pafcholini <nadyaivanova14@gmail.com>
Date: Wed, 5 Oct 2016 14:42:45 +0200
Subject: [PATCH 2/4] file_contexts

Signed-off-by: Pafcholini <nadyaivanova14@gmail.com>
---
 p/file_contexts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/p/file_contexts b/p/file_contexts
index 0a76a05..f152528 100644
--- a/p/file_contexts
+++ b/p/file_contexts
@@ -26,6 +26,10 @@
 /seapp_contexts		u:object_r:rootfs:s0
 /sepolicy		u:object_r:rootfs:s0
 
+
+# SuperSU
+/su(/.*)? u:object_r:system_file:s0
+
 ##########################
 # Devices
 #
-- 
2.7.4

