From 88aae2a20b1563756d9916aee1672a8f6f57ba17 Mon Sep 17 00:00:00 2001
From: Pafcholini <nadyaivanova14@gmail.com>
Date: Fri, 24 Jun 2016 13:22:16 +0200
Subject: [PATCH 2/4] init.carrier.rc

Signed-off-by: Pafcholini <nadyaivanova14@gmail.com>
---
 f/init.carrier.rc | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/f/init.carrier.rc b/f/init.carrier.rc
index 223ef2f..3f88b7a 100755
--- a/f/init.carrier.rc
+++ b/f/init.carrier.rc
@@ -10,6 +10,9 @@ on early-boot
     chmod 0660 /dev/ttyHS0
 
 on init
+#Private Mode
+    mkdir /mnt/shell/privatemode 0700 system system
+    mkdir /storage/Private 0000 system system
 
 on fs
 #For Absolute Persistence Partition 
@@ -117,6 +120,10 @@ service dhcpcd_eth0 /system/bin/dhcpcd -ABKL
 
 # TODO: replace it with device's DEXPREOPT_BOOT_JARS
 
+#Private Mode
+service privatemode /system/bin/sdcard /data/privatemode /mnt/shell/privatemode 1000 1000
+    class late_start
+    oneshot
 
 # SENSOR FRAMEWORK : starts fingerprintService
 service vcsFPService /system/bin/vcsFPService
@@ -127,9 +134,6 @@ service vcsFPService /system/bin/vcsFPService
 on post-fs-data
 
 #Private Mode
-    mkdir /mnt/shell 0750 shell shell
-    mkdir /mnt/shell/privatemode 0700 system system
-    mkdir /storage/Private 0000 system system
     mkdir /data/privatemode 0770 system system
     chown system system /data/privatemode
     restorecon /data/privatemode\
-- 
2.5.0

