From 23692ad073745f24e77c0e106c9d4395f45cea59 Mon Sep 17 00:00:00 2001
From: Pafcholini <nadyaivanova14@gmail.com>
Date: Fri, 24 Jun 2016 13:25:38 +0200
Subject: [PATCH 3/4] init.qcom.rc

Signed-off-by: Pafcholini <nadyaivanova14@gmail.com>
---
 f/init.qcom.rc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/f/init.qcom.rc b/f/init.qcom.rc
index 1164408..38e0489 100755
--- a/f/init.qcom.rc
+++ b/f/init.qcom.rc
@@ -27,7 +27,6 @@
 
 import init.qcom.usb.rc
 import init.target.rc
-import init.carrier.rc
 
 on early-init
     mount debugfs debugfs /sys/kernel/debug
@@ -239,6 +238,8 @@ on post-fs-data
     # we will remap this as /mnt/sdcard with the sdcard fuse tool
     mkdir /data/misc/camera 0770 camera camera
 
+    mkdir /data/media 0770 media_rw media_rw
+    chown media_rw media_rw /data/media
 
     mkdir /data/misc/bluetooth 0770 bluetooth bluetooth
 
-- 
2.5.0

